packages/graphql/tests/integration/deprecations/aggregations/field-level/field-level-aggregations-graphql-alias.int.test.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { UniqueType } from "../../../../utils/graphql-types";
+import { TestHelper } from "../../../../utils/tests-helper";
+
+describe("Field Level Aggregations Graphql alias", () => {
+    const testHelper = new TestHelper();
+    let typeDefs: string;
+
+    let typeMovie: UniqueType;
+    let typeActor: UniqueType;
+
+    beforeAll(async () => {
+        typeMovie = testHelper.createUniqueType("Movie");
+        typeActor = testHelper.createUniqueType("Actor");
+
+        typeDefs = `
+        type ${typeMovie.name} @node {
+            title: String
+            ${typeActor.plural}: [${typeActor.name}!]! @relationship(type: "ACTED_IN", direction: IN, properties:"ActedIn")
```

Comment:
```suggestion
            actors: [${typeActor.name}!]! @relationship(type: "ACTED_IN", direction: IN, properties:"ActedIn")
```
This is applicable also below